tests/utils/factories.py

```diff
@@ -4,20 +4,20 @@
 
 from cms.models import Placeholder
 
-import factory
 from djangocms_versioning.models import Version
 from djangocms_versioning.test_utils.factories import (
     AbstractVersionFactory,
     PageVersionFactory,
 )
-from factory.fuzzy import FuzzyChoice, FuzzyInteger, FuzzyText
 
+import factory
```

Comment:
Factory should be at the top of the file. They were done wrong before. 

```
import string
import factory

from djangocms_versioning.models import Version
from djangocms_versioning.models import Version
from factory.fuzzy import FuzzyChoice, FuzzyInteger, FuzzyText

from .moderated_polls.models import Poll, PollContent, PollPlugin

```

Reply:
isort would complain if I moved it to that location. So from what I can tell it follows the principle of grouping things according to stdlib, 3rd party and package imports.